src/components/table/tableWrappers/citiesTableWrappers/CitiesTable.tsx

```diff
@@ -14,15 +14,20 @@ import { City } from "@/entities/City";
 import { deserializeOrdinanceMetadata } from "../../fetchFunctions/loadOrdinanceMetadata";
 import LinkButton from "../../../buttons/LinkButton";
 import { Badge } from "@tremor/react";
+import { SchoolTypeValues } from "@/entities/School";
 
 export default function CitiesTable({
   initialData,
   newOrdinanceMetadata,
   count,
+  schoolType,
+  rootPath,
 }: {
   initialData: any[];
   newOrdinanceMetadata: any[];
   count?: number;
+  schoolType: string;
```

Comment:
Same thing with the `schoolType` type and the redundancy of `rootPath` param here.